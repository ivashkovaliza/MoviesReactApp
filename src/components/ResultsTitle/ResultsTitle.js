import React from "react";
import PropTypes from 'prop-types';
import './ResultsTitle.scss';

const ResultsTitle = (props) => { 
  return <p>{props.title}</p>
};

ResultsTitle.propTypes = {
    title: PropTypes.string,
};

export default ResultsTitle;
