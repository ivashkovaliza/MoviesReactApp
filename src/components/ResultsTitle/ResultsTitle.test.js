import React from 'react';
import ResultsTitle from './ResultsTitle';

describe('ResultsTitle', () => {
  test('renders correctly', () => {
    const component = shallow(
      <ResultsTitle />
    );
    expect(component).toMatchSnapshot();
  });
});
