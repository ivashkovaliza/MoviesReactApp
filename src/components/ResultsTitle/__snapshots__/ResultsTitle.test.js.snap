// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsTitle renders correctly 1`] = `<p />`;
